### ISSUE TITLE
Tabindex missing from edit settings link

### ISSUE THREAD

**Issue by danieltj27** [danieltj27/Phost#21]
The faux-link to close the editor settings panel is keyboard focusable as it's missing a tabindex attribute and also a jQuery click event so isn't accessible for keyboard users.

**Comment by danieltj27** [comment]
Forgot to mention in commit message but this is fixed in commit c31a301.
